### DISCUSSION TITLE
翻译文档 migration/schedule.md

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#423] [MMagic]
# 翻译文档 migration/schedule.md 

翻译[https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/migration/schedule.md](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/migration/schedule.md) 

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：tackhwa

**Claim Time**：2023年07月11日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1937

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by tackhwa** [comment]
领取任务

**Comment by tackhwa** [comment]
https://github.com/open-mmlab/mmagic/pull/1937

**Reply by crazysteeaam** [reply]
您好，tackhwa同学~
您之前完成的任务[https://github.com/open-mmlab/OpenMMLabCamp/discussions/423](https://github.com/open-mmlab/OpenMMLabCamp/discussions/423)已经升级为了超级视客营任务，完成任务后会享受更丰富的积分体系和好礼~

现在需要您登录[https://openmmlab.com/activity/codecamp](https://openmmlab.com/activity/codecamp)，按照流程申请一下这个任务，我们会直接为您无门槛通过任务审核~
同时请您加一下微信号：OpenMMLabwx，方便联系您！

**Comment by crazysteeaam** [comment]
good job

tackhwa ***@***.***>于2023年7月13日 周四23:22写道：
